_posts/2018-10-16-meetup.md

```diff
@@ -0,0 +1,58 @@
+---
+title: An Event-ful Evening with JavaScript
+author: lon
+layout: post
+categories:
+- posts
+- meetups
+when: 2018-10-16T19:30:00-05:00
+---
+
+{% assign speakr = 'Marie Chatfield' %}
+{% assign twiturl = 'https://twitter.com/mariechatfield' %}
+{% assign huburl = 'https://www.github.com/mariechatfield' %}
+
+Handling events like clicks and keypresses as users interact with your content
+is essential for any web app. Learn the fundamentals of how events work in
+JavaScript, including:
+
+* an introduction to the DOM Event specification
+* how to attach event listeners (there's so many ways!)
+* how event propagation works, what "bubbling" and "capturing" mean, and why you
+should care
+* the differences between preventDefault, stopPropagation, and return false (and
+when you should use each one!)
+
+Whether you're just learning JavaScript or are an experienced front-end
+developer, you'll walk away with a deeper familiarity with events. You'll be
+able to implement event listeners with confidence and debug with clarity!
+
+This talk is adapted from an Ember.js-specific talk, [Deep Dive on Ember
+Events][].
+
+### Our speaker
+
+<div class="media-object speaker-bio">
+  <a href="{{ twiturl }}">
+    <img alt="{{ speakr }} @mariechatfield on Twitter"
+      src="https://pbs.twimg.com/profile_images/900229466852655104/TeYmhngm_400x400.jpg" />
+  </a>
+  <div>
+  <a href="{{ twiturl }}"><strong>{{ speakr }}</strong></a>
+
+  writes code and poetry. loves jesus. always looking for new things to learn.
+  http://my.pronoun.is/she/her 
```

Comment:
Updated speaker bio:

Marie Chatfield is a software engineer at [Pingboard](https://pingboard.com/). A native Houstonian, Marie has recently returned to the homeland after a three-year stint in San Francisco, where she consistently got into arguments by insisting that tacos are superior in every way to burritos. Marie is passionate about creating inclusive spaces, building empathetic and elegant software, learning new things, and turning herself into emojis and cardboard cutouts.